src/mantarray_desktop_app/constants.py

```diff
@@ -231,6 +231,8 @@
 SECONDS_TO_WAIT_WHEN_POLLING_QUEUES = 0.02  # Due to the unreliablity of the .empty() .qsize() methods in queues, switched to a .get(timeout=) approach for polling the queues in the subprocesses.  Eli (10/26/20): 0.01 seconds was still causing sporadic failures in Linux CI in Github, so bumped to 0.02 seconds.
 
 # Serial Communication Values
+SERIAL_COMM_BAUD_RATE = 4000000
```

Comment:
note for future: typically slightly easier to read as `4e6` or whatever the exact syntax is--rather than counting a large number of zeros